src/leds.c

```diff
@@ -0,0 +1,73 @@
+#include "leds.h"
```

Comment:
better prefix all functions with led_* or better `oca_led_*` then you know the scope.